src/js/components/replenishment/CreateReplenishment.jsx

```diff
@@ -145,17 +145,24 @@ const FIELD = {
 function validate(values) {
   const errors = {};
   errors.requirements = [];
-
   _.forEach(values.requirements, (item, key) => {
+    // If item not checked, we don't want to validate
+    if (!item.checked) {
+      return;
+    }
+    // If item checked, validate
     if (item.quantity) {
```

Comment:
I think we can go even one step further and do something like:
```
if (!item.checked) {
  return;
}
if (!item.quantity) {
  errors.requirements[key] = { quantity: 'react.replenishment.error.emptyQtyToTransfer.label' };
  return;
}
if (item.quantity < 1) {
  errors.requirements[key] = { quantity: 'react.replenishment.error.quantity.label' };
}
if (item.quantity > item.quantityAvailable) {
  errors.requirements[key] = { quantity: 'react.replenishment.error.quantity.greaterThanQATP.label' };
}
```

We could also short circuit _(add **return** inside of each **if** to not go further because error is already found)_ but it is optional in this case